Transbank/Transbank.csproj

```diff
@@ -109,7 +109,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Web.Services3" Version="3.0.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3-beta1" />
```

Comment:
Why you shoul change version to a beta one?

Reply:
guess this is an unwanted change!